modules/identify-vulnerabilities-in-nodes-vm20.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/common-vuln-management-tasks.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="identify-vulnerabilities-in-nodes-vm20_{context}"]
+= Viewing Node CVEs
+
+You can use {product-title-short} to identify vulnerabilities in your nodes.
+
+The identified vulnerabilities include vulnerabilities in core Kubernetes components and container runtimes such as Docker, CRI-O, runC, and containerd. For more information about operating systems that {product-title-short} can scan, see "Supported operating systems".
+
+.Procedure
+. In the {product-title-short} portal, go to *Vulnerability Management* -> *Node CVEs*. You have the following options for viewing the data:
+* Select *<number> CVEs* to view a list of all the CVEs affecting all of your nodes. You can filter by node, CVE, node component, and cluster.
```

Comment:
@kcarmichael08 @charmik-redhat @dvail I was wondering if we want to specify that these filters are a bit different compared to the traditional/old-style filters that users have been using. I know the Advanced Filters work is mostly a subset of work done within VM 2.0 and Compliance 2.0, but the functionality is a bit different. The following is a brief description of the differences:

1. We display 2 dropdowns. The first displays the resource/entity (ie. Namespace, Deployment, CVE, etc.). The second displays some attribute/search value (ie. Name, Labels, Annotations, CVSS, etc.).
2. Choosing both will show the appropriate search filter input. The options being of of these: a text input, autocomplete text input, date picker, checkbox select, or a conditional number selector (ie. for CVSS).
3. Initiating a search will update the URL, add search filter chips in the UI, and filter table results.

I don't know how much we want to describe the improvements for filtering, but I think at least something would be nice and show that some extra work was done for this section in regards to improving search capabilities.

This would probably apply to each area where we mention the filters or it's own dedicated space. Whatever is more appropriate.

Reply:
It looks like you did mention the filters in a later section. You can disregard this then!

Reply:
Good idea - I added the ones available in the Node CVEs view to avoid users having to search/scroll.